package.json

```diff
@@ -2,10 +2,16 @@
     "name": "scope-web",
     "@comment dependencies": [],
     "dependencies": {
+        "@material-ui/core": "^4.11.3",
```

Comment:
I was using the `.x` minor syntax, you're using the `^` minor syntax.

They mean the same thing, `^` has been around longer, `.x` is easier for me to remember what it means.

So `.x` would be my preference from a usability perspective. 

You?

Reply:
I'll move this out into an issue so it's not delaying merge.